pint_server/app.py

```diff
@@ -691,7 +691,11 @@ def trim_images_payload(images):
 
 
 def get_provider_images(provider):
-    images = PROVIDER_IMAGES_MODEL_MAP[provider].query.order_by(
+    images = PROVIDER_IMAGES_MODEL_MAP[provider].query.filter(
+            PROVIDER_IMAGES_MODEL_MAP[provider].state.in_([
```

Comment:
Maybe use `not_in() instead of `in()` as we only want to exclude the deleted ones? In case we add more states in the future.

Reply:
Don't care either way ;-)

Reply:
I like the positive test, i.e. `in`

Reply:
So just tested and there isn't a `not_in` operator available for the `state` field as it is... Using the `not_in()` would require additional plumbing to import it and structure the code to leverage it, so sticking with the `in_()` is the better approach for now.